example/aiur.config.js

```diff
@@ -20,6 +20,15 @@ exports.vendor = {
 	scripts: []
 };
 
+exports.renderers = {
```

Comment:
add languages like this?
with option to add configuration

with / without configuration: 
```
exports.renderers = {
    handlebars: require("aiur-renderer-handlebars"),
    lymetheaf: [
        require("..."),
        { ... config}
    ]
}
```